build/scripts/build-utils.ps1

```diff
@@ -183,22 +183,27 @@ function Ensure-SdkInPathAndData() {
         Write-Host "Downloading CLI $sdkVersion"
         Create-Directory $cliDir
         Create-Directory $toolsDir
-        $webClient = New-Object -TypeName "System.Net.WebClient"
         if (Is-Unix) {
-            $destFile = Join-Path $toolsDir "dotnet-install.sh"
-            $webClient.DownloadFile("https://dot.net/v1/dotnet-install.sh", $destFile)
             # AppImage mucks with LD_LIBRARY_PATH and causes the install to fail
             $old_ld_path = $env:LD_LIBRARY_PATH
             try {
                 $env:LD_LIBRARY_PATH = ""
-                Exec-Block { & bash $destFile -Version $sdkVersion -InstallDir $cliDir } | Out-Null
+                $destFile = Join-Path $toolsDir "dotnet-install.sh"
+                # Should be using System.Net.WebClient.DownloadFile, but that fails
+                # on WSL with "System.Net.Http.CurlException: Couldn't resolve host name"
+                Exec-Block { & curl "https://dot.net/v1/dotnet-install.sh" -o $destFile } | Out-Null
```

Comment:
Just a warning, `curl` may not be calling `curl` at all and might be calling a PowerShell alias (depending on the version you have, I believe it is fixed in the latest, but I am not positive).

Reply:
What do you suggest instead?

Reply:
If the alias issue is fixed in this version of powershell, it might not be an issue.

Otherwise, you can use `Invoke-WebRequest` (https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/invoke-webrequest?view=powershell-5.1). Noting that you must explicitly pass in `-UseBasicParsing` or can have failures on Windows when IE is not installed or configured.

Or you can find out why `WebClient` isn't working. I know WSL and Azure Linux VMs sometimes don't have the localhost name configured correctly and can cause that error. Checking that `/etc/hostname` contains the name of the machine and `/etc/hosts` contains an entry like `127.0.0.1 localhost localhost.localdomain` and `127.0.1.1 <hostname>`

Reply:
`Invoke-WebRequest` fails with the same issue.

`/etc/hostname` and `/etc/hosts` are sane/valid, and normal `curl` works fine.

Also, creating a new dotnet console app and creating/using a WebClient from C# works fine.

No idea, heh.

Reply:
It sounds like a potential bug in the WebClient implementation then. We should probably log something on dotnet/corefx.